### PR TITLE
PSR 287

### PR DESCRIPTION
**Pull request by EricSiegNW**
add some detail text handling for insights and withdraw profile items

### CODE REVIEW

File: Psorcast/Psorcast/PopTip/PopTipController.swift [hunk 1]
```diff
@@ -137,8 +137,6 @@ public class PopTipController: ShowPopTipDelegate {
             if let psoriasisDrawStepViewController = viewController as? PsoriasisDrawStepViewController {
                 let buttonFrame = psoriasisDrawStepViewController.undoButton.frame
                 let popTipRect = buttonFrame.offsetBy(dx: 0, dy: -10)
-                // scrollview offset y
```
Comment (EricSiegNW) [inline Psorcast/Psorcast/PopTip/PopTipController.swift:140]: cleanup

**Comment by EricSiegNW** [comment]
Most of the work is done by adjusting the profile data source json in bridge, this just adds some profile item handling to add detail text for withdraw (not enrolled NYI) and Insights

File: Psorcast/Psorcast/ProfileTabViewController.swift [hunk 2]
```diff
@@ -98,6 +98,22 @@ class ProfileTabViewController: UIViewController, UITableViewDelegate, UITableVi
...
+        } else if (self.historyData.pastInsightItemsViewed.count == 1) {
+            return Localization.localizedString("FIRST_INSIGHT_UNLOCKED")
+        } else {
+            return String(format: Localization.localizedString("%@_INSIGHTS_ACHIEVED"), "\(self.historyData.pastInsightItemsViewed.count)")
+        }
+    }
+    
+    func withdrawProfileItemValue() -> String {
+        return Localization.localizedString("CURRENTLY_ENROLLED")
+        // Todo: add logic to determine if enrolled, adjust return values here appropriately
```
Comment (EricSiegNW) [inline Psorcast/Psorcast/ProfileTabViewController.swift:114]: Would they make it to this screen without being enrolled?

Comment (DanWebster) [reply Psorcast/Psorcast/ProfileTabViewController.swift:114]: @EricSiegNW good question, which I think the answer is no.  Still, it doesn't hurt to have this little reminder that you are in a research study currently

**Review by DanWebster — CHANGES_REQUESTED** [review]
@EricSiegNW I couldn't find the strings in Psorcast.strings or in the profile view controller to be able to make a quick committable change here; perhaps this is specified in the storyboard? Could you change the tableviewcell title text for the 'Health Information' section to be the following:
Psoriasis diagnosis -> Diagnoses
Psoriasis Symptoms -> Current Symptoms

**Review by DanWebster — APPROVED** [review]

**Pull request merged**